src/web/mjs/engine/Connector.mjs

```diff
@@ -419,6 +419,7 @@ export default class Connector {
         if( clearIframettributes ) {
             content = content.replace( /<iframe[^<]*?>/g, '<iframe>');
         }
+        content = content.replace(/<\/?noscript>/g, '');
```

Comment:
I can see the removal of `noscript` tags, but why is it needed?

Reply:
Any HTML tags inside the `noscript` tags are not parsed as DOM nodes, instead `noscript` is a text node containing the plain HTML code. In short one of the website used this and to counter it the `noscript` wrapper is removed to expose the contained HTML text and parse it as DOM nodes.